modules/javafx.graphics/src/main/java/javafx/css/StyleablePropertyFactory.java

```diff
@@ -141,7 +143,7 @@ public final class MyButton extends Button {
 
      // StyleableProperty implementation reduced to one line
      private final {@literal StyleableProperty<Boolean>} selected =
-         new SimpleStyleableBooleanProperty(this, "selected", "my-selected");
+         new SimpleStyleableBooleanProperty(SELECTED, "selected", "my-selected");
```

Comment:
as an exercise, could you actually put MyButton into a scene graph and style it with CSS?

Reply:
I will check this and update here

Reply:
Do you think it's worth doing this as a unit test?

Reply:
I think it is good to add unit test to test the example code. Only thing I'm looking at is that if the example changes in future, we need to make sure that corresponding `MyButton` class is updated in unit test. 
Since the example code might not change frequently, we can add unit test.

Reply:
I agree, let's add a unit test, unless  `StyleablePropertyFactoryTest` covers the same scenario.

Reply:
Added unit test for the 3rd example in `StyleablePropertyFactoryTest`.
Unit test for 1st example is already covered as part of `myBoolean` property.
Since example 2 and 3 are similar except for the place where `createBooleanCssMetaData` method is called, added single test for this.
Please review the newly added unit test